packages/forms/docs/stories/UniversalFormOverlay.story.js

```diff
@@ -0,0 +1,72 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { Field } from 'react-final-form';
+import { Input, ButtonIcon } from 'react-rainbow-components';
+import { RainbowFirebaseApp } from '@rainbow-modules/app';
+import { Plus } from '@rainbow-modules/icons';
+import UniversalFormOverlay from '../../src/components/UniversalFormOverlay';
+import isRequired from '../../src/validators/isRequired';
+import app from '../../../../firebase';
+
+const TitleContainer = styled.div`
+    margin: 20px 50px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const Title = styled.h1`
+    font-size: 30px;
+    color: ${(props) => props.theme.rainbow.palette.text.main};
+    font-family: Lato Bold, Helvetica, sans-serif;
+`;
+
+const Label = styled.span`
+    font-size: 1rem;
+    font-weight: bold;
+`;
+
+const Fields = () => {
+    return (
+        <div>
+            <Label>Name</Label>
```

Comment:
I think we should use the label prop of the input, check with @TahimiLeonBravo if the styles need to be customized, then you can pass a node to the label prop